src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,59 @@
+package lotto.domain;
+
+import lotto.constant.LottoRanking;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class LottoResult {
+    private final Map<Integer, Integer> lottoRankings = new HashMap<>();
+    private final int MIN_RANKING = 3;
+    private final int MAX_RANKING = 6;
```

Comment:
```suggestion
    private final Map<LottoRanking, Integer> lottoRankings = new HashMap<>(); // (1)
    private static final int MIN_RANKING = 3; // (2)
    private static final int MAX_RANKING = 6;
```
(1) Ranking을 키로 잡아도 될것 같고, 그게 아니면 (2) 상수로 만들어주면 좋을 것 같습니다ㅎㅎ